snuba/datasets/storages/transactions_common.py

```diff
@@ -68,10 +68,8 @@
         ("http_method", String(Modifiers(nullable=True))),
         ("http_referer", String(Modifiers(nullable=True))),
         ("tags", Nested([("key", String()), ("value", String())])),
-        ("_tags_flattened", String()),
```

Comment:
Does this make deploy complicated? e.g. will we cause errors if the ColumnSet is missing columns returned from the database? Or is it safe to deploy this logic change and then run the migrations later?

Reply:
I don't think you have to worry about prod since we don't run migrations automatically. Less certain about single-tenant/open source

Reply:
The reason I would separate these out is that if the consumer is broken because of the change, I just need to add the columns back. If I do it this way, the migrations would need to be reverted and I would want to avoid reverting migrations.